scripts/domaintools.js

```diff
@@ -1,85 +1,38 @@
-                parent.appendChild(child);
-            }
-            if(data.response.results[0].registrant_org.value){
-                var child=document.getElementById("registrant_org");
-                child.innerHTML="Registrant Org: "+data.response.results[0].registrant_org.value;
-                parent.appendChild(child);
-            }
-            if(data.response.results[0].website_response){
-                var child=document.getElementById("website_response");
-                if(data.response.results[0].website_response==200){
-                    child.innerHTML="Website Response Status Code: "+data.response.results[0].website_response+" OK";
-                }else if(data.response.results[0].website_response==404){
-                    child.innerHTML="Website Response Status Code: "+data.response.results[0].website_response+" Not Found";
-                }
-                parent.appendChild(child);
-            }
-            if(data.response.results[0].whois_url){
-                var child=document.getElementById("whois");
-                child.href=data.response.results[0].whois_url;
-                child.innerHTML="Click to see the Whois URL";
-                parent.appendChild(child);
-            }
-        }else{
-            document.getElementById("show_domaintools_data").innerHTML="No data found!!";
+function url_getter () {
+  var url = getUrlByParameter('url')
+  var domaintools_api = 'https://archive.org/services/context/domaintools?url=' + url
+  $.getJSON(domaintools_api, function (data) {
+    var parent = $('#show_domaintools_data')
+    if (data.response.results_count != 0) {
+      appendToParent('#domain', data.response.results[0].domain, 'Domain: ', parent, true, '')
+      appendToParent('#alexa', data.response.results[0].alexa, 'Alexa Rank: ', parent, true, '')
+      appendToParent('#admin_contact_country', data.response.results[0].admin_contact.country.value, 'Country: ', parent, true, '')
+      appendToParent('#create_date', data.response.results[0].create_date.value, 'Created Date: ', parent, true, '')
+      appendToParent('#email_domain', data.response.results[0].email_domain[0].value, 'Email Domain: ', parent, true, '')
+      appendToParent('#expiration_date', data.response.results[0].expiration_date.value, 'Expire Date: ', parent, true, '')
+      appendToParent('#admin_contact_state', data.response.results[0].admin_contact.state.value, 'State: ', parent, true, '')
+      appendToParent('#registrant_org', data.response.results[0].registrant_org.value, 'Registrant Org: ', parent, true, '')
+      if (data.response.results[0].website_response) {
+        if (data.response.results[0].website_response == 200) {
+          appendToParent('#website_response', data.response.results[0].website_response, 'Website Response Status Code: ', parent, true, ' OK')
+        } else if (data.response.results[0].website_response == 404) {
```

Comment:
Isn't it possible to have any other status code besides 200 and 404? There are so many status codes...
Can you also add a case to just display `Website Response Status Code: XXX`?